app/dts/bindings/behaviors/zmk,behavior-hold-tap.yaml

```diff
@@ -20,8 +20,11 @@ properties:
     default: -1
   quick_tap_ms: # deprecated
     type: int
-  global-quick-tap:
+  global-quick-tap: # deprecated
```

Comment:
I wasn't aware of this previously, but this should be using the dedicated `deprecated` property for bindings to properly warn on this one: https://docs.zephyrproject.org/latest/build/dts/bindings-syntax.html#deprecated

Reply:
Oh nice, I went ahead and labeled the other "deprecated" params as well, hopefully that's okay.